> pyenv root
> conda --version
> python3.7 -c "import sys;print(sys.executable)"
> python3.6 -c "import sys;print(sys.executable)"
> python3 -c "import sys;print(sys.executable)"
> python2 -c "import sys;print(sys.executable)"
> python -c "import sys;print(sys.executable)"
> /sbin/python -c "import sys;print(sys.executable)"
> conda info --json
Python interpreter path: /sbin/python
Starting Pylance language server.
Error 2021-09-10 19:58:57: Failed to check if file needs to be fixed [EntryNotFound (FileSystemError): Unable to read file '/home/mwalk/Documents/hugo-blog/.vscode/settings.json' (Error: Unable to resolve non-existing file '/home/mwalk/Documents/hugo-blog/.vscode/settings.json')
	at _handleError (/opt/visual-studio-code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:84:167834)
	at processTicksAndRejections (internal/process/task_queues.js:93:5)
	at async y.readText (/home/mwalk/.vscode/extensions/ms-python.python-2021.8.1159798656/out/client/extension.js:9:320331)
	at async p.doesFileNeedToBeFixed (/home/mwalk/.vscode/extensions/ms-python.python-2021.8.1159798656/out/client/extension.js:59:586718)
	at async /home/mwalk/.vscode/extensions/ms-python.python-2021.8.1159798656/out/client/extension.js:59:585845
	at async Promise.all (index 1)
	at async p.getFilesToBeFixed (/home/mwalk/.vscode/extensions/ms-python.python-2021.8.1159798656/out/client/extension.js:59:585791)
	at async p.updateTestSettings (/home/mwalk/.vscode/extensions/ms-python.python-2021.8.1159798656/out/client/extension.js:59:585418)] {
  code: 'FileNotFound'
}
> pyenv root
> conda --version
